pkg/network/master/master.go

```
@@ -146,6 +146,27 @@ func Start(networkConfig osconfigapi.MasterNetworkConfig, networkClient networkc
 		return err
 	}
 
+	var nodes []kapi.Node
+	var subnets []networkapi.HostSubnet
+	if nodeList, err := master.kClient.Core().Nodes().List(metav1.ListOptions{}); err == nil {
```

Do not ignore the error
```
subnets, err := master.kClient.Core().Nodes().List(metav1.ListOptions{})
if err != nil {
   //throw error
}
for _, subnet := range subnets.Items {
...
}
```